libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -127,6 +127,14 @@ internal class MapboxTripSession(
             offRouteObservers.forEach { it.onOffRouteStateChanged(value) }
         }
 
+    private var withTripService: Boolean = true
+        set(value) {
```

Comment:
Is this setter really needed?

Reply:
I used it just for consistency in this class 🤷🏻

Reply:
setter is no matter here, lets remove it